src/Verification.php

```diff
@@ -28,11 +31,19 @@ public function __construct($message, KeyStoreInterface $keyStore)
     /**
      * @return bool
      */
-    public function isValid()
+    public function isSigned()
```

Comment:
This is a BC break. Could we keep (and deprecate) the `isValid` method instead?

Reply:
Unsure what the strategy would be, advise?
Similarly the existing `sign()` generates `Signature` and `Authorization`, so generating only `Signature` would be a BC

Reply:
keep `isValid` function around, mark it as deprecated and `return isSigned && isAuthorized`

Reply:
I can re-introduce `isValid()`, but don't know the approach to deprecate. Is this a hint, a warning emitted in runtime, documentation only?

Reply:
@mtibben Any advice?